inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/mq/BatchPackProfile.java

```diff
@@ -63,36 +54,17 @@ public BatchPackProfile(String uid, String inlongGroupId, String inlongStreamId,
      * @param  maxPackSize
      * @return
      */
-    public boolean addEvent(ProxyEvent event, long maxPackCount, long maxPackSize) {
+    public boolean addEvent(Event event, long maxPackCount, long maxPackSize) {
         long eventLength = event.getBody().length;
         if (count >= maxPackCount || (count > 0 && size + eventLength > maxPackSize)) {
             return false;
         }
-        this.events.add(event);
+        this.events.add((ProxyEvent) event);
```

Comment:
Suggest events  type from List<ProxyEvent> events  to  List<Event> events

Reply:
The call side of this API is passed event by event. It essentially wants to accumulate a certain number and size before sending it. For this PR，only the type change is made, and then the logical level change is made later.